### PR TITLE
Add q to quit out of menus

### PR DESCRIPTION
**Pull request by WillNilges**
wip. Need to add this to like, every menu.

### CODE REVIEW

File: src/ui/inventory.rs [hunk 1]
```diff
@@ -109,6 +109,10 @@ pub fn build_menu(api: &mut api::API, machine_status: &Value, machine_index: i32
               ui_common::destroy_win(win);
               return;
             },
+            113 => {
```
Comment (Mstrodl) [inline src/ui/inventory.rs:112]: Are there not constants for these things?

**Review by Mstrodl — CHANGES_REQUESTED** [review]

**Comment by Mstrodl** [comment]
:scream: `.swp` file!

**Comment by WillNilges** [comment]
shut the fuck up it was like 4AM and I wanted to push something before I got out of the drink closet leave me alone

File: src/ui/inventory.rs (hunk 1 above)
Comment (WillNilges) [reply src/ui/inventory.rs:112]: there are in a commit i haven't pushed yet.

**Comment by WillNilges** [comment]
DO NOT REVIEW! I am still working on this.

**Comment by Mstrodl** [comment]
Fixed now

**Pull request closed without merging**